deployments/apprentice-commitments.yml

```diff
@@ -0,0 +1,228 @@
+trigger: none
+pr: none
+
+variables:
+- group: Release Management Resources
+- group: RELEASE das-apim-endpoints
+- name: OuterApiObject
+  value: $(ApprenticeCommitmentsOuterApiObject)
+
+resources:
+  repositories:
+  - repository: self
+  - repository: das-platform-building-blocks
+    type: github
+    name: SkillsFundingAgency/das-platform-building-blocks
+    ref: refs/tags/0.3.14
+    endpoint: SkillsFundingAgency
+  - repository: das-platform-automation
+    type: github
+    name: SkillsFundingAgency/das-platform-automation
+    ref: refs/tags/4.4.1
+    endpoint: SkillsFundingAgency
+  pipelines:
+  - pipeline: das-apim-endpoints-build
+    project: Digital Apprenticeship Service
+    source: das-apim-endpoints (build)
+    branch: master
+  - pipeline: das-employer-config
+    project: Digital Apprenticeship Service
+    source: das-employer-config
+    branch: master
+
+stages:
+- stage: Deploy_AT
+  dependsOn: []  
+  displayName: Deploy to AT
+  variables:
+  - group: DevTest Management Resources
+  - group: AT DevTest Shared Resources
+  - group: AT das-apim-endpoints
+  jobs:
+  - template: ../pipeline-templates/job/arm-deploy.yml
```

Comment:
Could we pass another parameter for schema file to process? So only the single Outer API's file is processed?

Reply:
Great suggestion. Will get this applied 👍